docs/DeveloperGuide.adoc

```diff
@@ -1142,6 +1142,105 @@ These instructions only provide a starting point for testers to work on; testers
 .. Re-launch the app by double-clicking the jar file. +
    Expected: The most recent window size and location is retained.
 
+=== Achievement Mechanism
+
+. Update of xp and number of tasks completed on task completion
+
+.. Test case: `complete 1` +
+Expected: Xp value on the achievement panel increases by amount reported by complete
+command message(determined by game mode).Tasks completed increases by 1.
+
+.. Test case: `complete l/tutorial` +
+Expected: Xp value on the achievement panel increases by amount reported by complete
+command message(determined by game mode). Tasks completed increases by the number of
+tasks completed by the batch complete command (as specified in the complete command message).
+
+. Update of level on task completion
+.. Prerequisites: change the specified fields in data/taskmanager.xml(or other storage path) to:
+`<xp>499</xp>` +
+`<level>lvl.1</level>` +
+`<xpValueByDay>0</xpValueByDay>` +
+`<xpValueByWeek>0</xpValueByWeek>` +
+.. Test case: `complete 1` +
+Expected: Xp value on the achievement panel increases by amount reported by complete
+command message(determined by game mode). Tasks completed increases by 1. And level increases to level reported by
+complete command message(determined by game mode).
```

Comment:
This clause makes it sound like complete command's output message is determined by game mode and that achievement panel's value is dependent on complete command's message. Iirc, both achievement panel and complete command's output value is determined by the achievement record

Reply:
changed!